src/main/java/com/epam/test_generator/dao/impl/JiraStoryDAO.java

```diff
@@ -70,32 +65,23 @@ public JiraStory getStoryByJiraKey(String jiraKey) throws JiraException {
         return issues.stream().map(JiraStory::new).collect(Collectors.toList());
     }
 
-    public void updateStoryByJiraKey(Suit suit) throws JiraException {
-        client
-                .getIssue(suit.getJiraKey())
-                .update()
-                .field(Field.SUMMARY, suit.getName())
-                .field(Field.DESCRIPTION, suit.getDescription())
-                .execute();
-        suit.setLastJiraSyncDate(LocalDateTime.now());
-        suitDAO.save(suit);
-    }
-
-    public void closeStoryByJiraKey(String jiraKey) throws JiraException {
-
+    public void changeStatusByJiraKey(String jiraKey, Integer action_id) throws JiraException {
+        if (action_id == null) {
```

Comment:
underscore in parameter names violates naming conventions: https://docs.oracle.com/javase/tutorial/java/nutsandbolts/variables.html